tssc/step_implementers/shared/openscap_generic.py

```diff
@@ -0,0 +1,656 @@
+            buildah_unshare_comand=buildah_unshare_comand,
+            oscap_eval_type=oscap_eval_type,
+            oscap_input_file=oscap_input_file,
+            oscap_out_file_path=oscap_out_file_path,
+            oscap_xml_results_file_path=oscap_xml_results_file_path,
+            oscap_html_report_path=oscap_html_report_path,
+            container_mount_path=container_mount_path,
+            oscap_profile=oscap_profile,
+            oscap_tailoring_file=oscap_tailoring_file,
+            oscap_fetch_remote_resources=oscap_fetch_remote_resources
+        )
+        print(
+            f"OpenSCAP Scan Completed.  Report: {oscap_html_report_path}\n"
+        )
+
+        # NOTE: this should not raise an exception once we have new Results object
+        #       since this is an "expected" "valid" failure...
+        #       but this is the solution for now
+        if not oscap_eval_success:
+            raise TSSCException(f"OSCAP eval found issues:\n{oscap_eval_fails}")
+
+        results = {
+            'result': {
+                'success': True
+            },
+            'report-artifacts': [
+                {
+                    'name': 'html-report',
+                    'path': f'file://{oscap_html_report_path}'
+                },
+                {
+                    'name': 'xml-report',
+                    'path': f'file://{oscap_xml_results_file_path}'
+                },
+                {
+                    'name': 'stdout-report',
+                    'path': f'file://{oscap_out_file_path}'
+                }
+            ]
+        }
+        return results
+
+    @staticmethod
+    def __buildah_import_image_from_tar(image_tar_file, container_name):
+        """Import a container image using buildah form a TAR file.
+
+        Parameters
+        ----------
+        image_tar_file : str
+            Path to TAR file to import as a container image.
+        container_name : str
+            name for the working container.
+
+        Returns
+        -------
+        str
+            Name of the imported container.
+        """
+        # import image tar file to vfs file system
+        try:
+            sh.buildah( # pylint: disable=no-member
+                'from',
+                '--storage-driver', 'vfs',
+                '--name', container_name,
+                f"docker-archive:{image_tar_file}",
+                _out=sys.stdout,
+                _err=sys.stderr,
+                _tee='err'
+            )
+        except sh.ErrorReturnCode as error:
+            raise RuntimeError(
+                f'Unexpected runtime error importing the image ({image_tar_file}): {error}'
+            ) from error
+
+        return container_name
+
+    @staticmethod
+    def __buildah_mount_container(buildah_unshare_comand, container_id):
+        """Use buildah to mount a container.
+
+        Parameters
+        ----------
+        buildah_unshare_comand : sh.buildah.unsahre.bake()
```

Comment:
Nit: 
`buildah_unshare_comand : sh.buildah.unsahre.bake()`

to 

`buildah_unshare_comand : sh.buildah.unshare.bake()`